From 0c2868c7e595c2a3dd61a9bcbbd882cb67b10d0f Mon Sep 17 00:00:00 2001
From: drosdeck <edson.drosdeck@gmail.com>
Date: Mon, 7 Nov 2022 12:14:07 -0300
Subject: [PATCH] Fix key toggle and programmable button for Positivo N14ZP

(cherry picked from commit 4dbcb5dc10fb8dd53202aa4ed3cff8c869ee3b4f)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 066a7a1c8f..60af8fde4d 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2022,6 +2022,8 @@ evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn
 
 # Positivo Motion (K142, K142-V2)
 evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnK142*
+# Positivo Motion (N14ZP6, N14ZP7)
+evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnN14ZP[67]*
 # Positivo Motion (CW14Q01P) (CW14Q01P-V2)
 evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnCW14Q01P*
  KEYBOARD_KEY_70073=f21                                 # Fn+f2 toggle touchpad
